crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -254,7 +254,8 @@ impl<'src> Parser<'src> {
         }
 
         // test_ok simple_stmts_with_semicolons
-        // return; import a; from x import y; z; type T = int
+        // def outer():
```

Comment:
nit: I think the function names don't matter here so would prefer to just keep it `_`